ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -197,7 +214,8 @@ const TransactionsScreen: React.FC<Props> = (props: Props) => {
 };
 
 const mapStateToProps = (state: GlobalState) => ({
-  favoriteWallet: getFavoriteWalletId(state),
+  favouriteWallet: favoriteWalletIdSelector(state),
```

Comment:
Could be a better name `favouriteWalletRequestStatus`, since is used to check if the request is loading?
```suggestion
  favoriteWallet: favoriteWalletIdSelector(state),
```

Reply:
changed in https://github.com/pagopa/io-app/pull/2930/commits/b3427893c23b94578344dbd9c858f0be9fa8e6c3